### PR TITLE
548 template meal plan

### LINKED ISSUE #548: [FEATURE] Template for mealplan
**Is your feature request related to a problem? Please describe.**
Use the same favorite icon for saving it as template. The difference between the template and normal meal plan is that the users will not be assigned.  I should be able to filter by templates.

**Describe the solution you'd like**
As a meal designer/admin I should be able to create a meal plan as a template with no user assigned. I should be able to filter by templates in the search view.

**Describe alternatives you've considered**
Favorites is the alternative, but does not make sense for meal plan
**Additional context**
Similar to favorites of meals in terms of implementation,.

### PR DESCRIPTION
**Pull request by priyaguntupalli5**
header
548 template
v2 template

**Describe the technical changes contained in this PR**
Modified CreateMealPlan and MealPlans files to allow meal designer and admin to create their own templates for Meal plans.

**Previous behaviour**
There was no option to create template for meal plans

**New behaviour**
Meal desginer/ admin can create templates with no user assigned

**Related issues addressed by this PR**
Fixes #548 

**Have the following been addressed?**
- [ ] Have test cases been created for all of the changes?
- [ ] Do all of the test cases pass?
- [ ] Has the testing been done using the default docker-compose environment?
- [ ] Are documentation changes required?
- [ ] Does this change break or alter existing behaviour?

### CODE REVIEW

File: mealplanner-ui/src/pages/MealPlans/CreateMealPlan.tsx [hunk 1]
```diff
@@ -102,139 +111,164 @@ export const CreateMealPlan = ({ connection, refetch }: { connection: string, re
...
+                  ></Autocomplete>
+                </Grid>
                 )}
-                onChange={(e, value) => {
-                  setUserId(value?.rowId);
-                }}
-              ></Autocomplete>
-            </Grid>
-            <Grid item xs={3}>
-              <Tooltip arrow placement="bottom-start" title={titleEn}>
```
Comment (shanthisa) [inline mealplanner-ui/src/pages/MealPlans/CreateMealPlan.tsx:125]: Why have you deleted this Tooltip and the rest of the code here?

File: mealplanner-ui/src/pages/MealPlans/CreateMealPlan.tsx [hunk 2]
```diff
@@ -102,139 +111,164 @@ export const CreateMealPlan = ({ connection, refetch }: { connection: string, re
...
-                  variant="filled"
-                  fullWidth
-                  onChange={(e) => {
-                    setDescriptionEn(e.target.value);
-                  }}
-                />
-              </Tooltip>
-            </Grid>
-            <Grid item xs={3}>
-              <Tooltip arrow placement="bottom-start" title={titleDescFr}>
```
Comment (shanthisa) [inline mealplanner-ui/src/pages/MealPlans/CreateMealPlan.tsx:168]: Looks like description Tooltip is also missing.

**Review by shanthisa — CHANGES_REQUESTED** [review]
Please provide screenshots of before your changes i.e. current v2 and your UI of Create Meal Plan / Template. Looks like you have missed Tool tips. Also I need to see how is the spacing and UI.

File: mealplanner-ui/src/pages/MealPlans/CreateMealPlan.tsx (hunk 2 above)
Comment (priyaguntupalli5) [reply mealplanner-ui/src/pages/MealPlans/CreateMealPlan.tsx:168]: Thanks for pointing this out. Fixed it and attached screenshots.

**Review by shanthisa — APPROVED** [review]
LGTM :+1

**Comment by shanthisa** [comment]
Ready for QA

**Comment by ala-sk98** [comment]
QA is Done.
Test case [MP-TemplateMealPlan-044](https://github.com/CivicTechFredericton/mealplanner/wiki/Test-case-portfolio-v1.0-and-2.0#test-case-id-MP-TemplateMealPlan-044): passed

**Comment by shanthisa** [comment]
@ala-sk98  This is a merge request. This test case should not be written as you have covered it in #548. Please remove this and provide this screenshot in the other test case written under #548. There should be no duplicate test cases

**Pull request merged**